application/templates/viewdeed.html

```diff
@@ -1,84 +1,86 @@
 {% extends "base/lr_base_layout.html" %}
 
 {% block content %}
-<div class="main" id="content">
+  <div class="main" id="content">
     {{ super() }}
 
     <h1 class="heading-xlarge">Your mortgage deed</h1>
 
     <div class="column-two-thirds">
-    <article class="deed-document-bounds">
+      <article class="deed-document-bounds">
         <h2 class="heading-large">Digital Mortgage Deed</h2>
-            <h3 class="heading-medium">Property </h3>
-                <p itemscope itemtype="http://schema.org/PostalAddress">
+        <h3 class="heading-medium">Property </h3>
+        <p itemscope itemtype="http://schema.org/PostalAddress">
                 <span itemprop="streetAddress">
                   Address Line 1<br>
                 </span>
-                <span itemprop="addressLocality">Locality</span><br>
-                <span itemprop="postalCode">PO57 CDE</span>
-              </p>
+          <span itemprop="addressLocality">Locality</span><br>
+          <span itemprop="postalCode">PO57 CDE</span>
+        </p>
 
-            <dl class="definition-inline">
-            <dt class="bold-text">Title number</dt>
-                <dd>{{ deed_data.deed.title_number }}</dd>
-            </dl>
+        <dl class="definition-inline">
+          <dt class="bold-text">Title number</dt>
+          <dd>{{ deed_data.deed.title_number }}</dd>
+        </dl>
 
-          <h3 class="heading-medium">Lender</h3>
+        <h3 class="heading-medium">Lender</h3>
         <div class="text">
-            Lender Information will go here
+          <p>{{ deed_data.deed.lender.name }}<br/>
```

Comment:
all 3 of these should probably be in their own paragraphs, semantically they are different things